src/components/Nav/Nav.scss

```
@@ -0,0 +1,99 @@
+.header {
+  display: flex;
+  min-width: 1260px;
+  height: 158px;
+  border-bottom: 1px solid #ededed;
+  .inner {
+    display: flex;
+    justify-content: space-between;
+    box-sizing: border-box;
+    position: sticky;
+    width: 1240px;
+    top: 0;
+    align-items: center;
+    margin: 0 auto;
+  }
+  .middle {
+    width: 400px;
+    display: flex;
+    align-items: flex-end;
+    height: 50px;
+    font-weight: 700;
+
+    .searchBar {
+      padding: 7px;
+      width: 450px;
+      border-radius: 10px;
+      border: 0.5px solid #6d5f5f;
+      font-size: 13px;
+    }
+    .searchIcon {
+      position: absolute;
+      left: 860px;
+      top: 78px;
+      height: 20px;
+    }
+
+    & ul {
+      & li {
```

```suggestion
     ul {
       li {
```
nesting 되어있어서 굳이 & 연산자는 사용하지 않아도 되겠습니다